services/eventservice/src/data_manager.rs

```diff
@@ -0,0 +1,304 @@
+use crate::main;
+
+use super::types::{Category, Event};
+use chrono;
+use serde::{de::IntoDeserializer, Deserialize, Serialize};
+use serde_json::value;
+use sled;
+use std::collections::HashSet;
+use tokio::runtime::Handle;
+
+pub struct EventManager {
+    db: sled::Db,
```

Comment:
what kind of database you are using?

Reply:
Some cool module I've found typed 'database' in cargo.io
sled's db is just set of Dictionary<char*[], char*[]>. It is interesting to work with, it is fully thread safe and pretty fast. From negative sides, it is pretty dumb and may use a bit more space than more popular dbs

Reply:
gotcha!